apps/user-office-frontend-e2e/cypress/integration/proposals.ts

```diff
@@ -120,6 +120,21 @@ context('Proposal tests', () => {
       });
     });
 
+    it('Copy to clipboard should work for Proposal ID ', () => {
```

Comment:
Nothing special but I just noticed a small detail here. There is one blank space in the end of the test title that could be removed.

Reply:
It is fixed. thx for pointing out that.